x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiTabbedContent, EuiSpacer, type EuiTabbedContentTab } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { MetricsGrid } from './metrics/metrics_grid';
+
+interface WrapperProps {
+  children: React.ReactElement;
+}
+const Wrapper = ({ children }: WrapperProps) => {
+  return (
+    <>
+      <EuiSpacer size="s" />
+      {children}
+    </>
+  );
+};
+export const Tabs = () => {
+  const tabs: EuiTabbedContentTab[] = [
+    {
+      id: 'metrics',
+      name: i18n.translate('xpack.infra.hostsViewPage.tabs.metricsCharts.title', {
+        defaultMessage: 'Metrics',
+      }),
+      'data-test-subj': 'hostsView-tabs-metrics',
+      content: (
+        <Wrapper>
+          <MetricsGrid />
+        </Wrapper>
+      ),
+    },
+  ];
+
+  return <EuiTabbedContent tabs={tabs} initialSelectedTab={tabs[1]} autoFocus="selected" />;
```

Comment:
**question:** should the property `initialSelectedTab` be `tabs[0]` since we only have an entry in the list of the tabs?

Reply:
Interesting that it still worked. It should be `tabs[0]`, for sure.